src/app/src/components/FacilitiesMap.jsx

```diff
@@ -1,519 +1,276 @@
-import React, { Component, Fragment } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import { bool, func, number, shape, string } from 'prop-types';
```

Comment:
The diff here isn't really meaningful -- this is an entirely new file, but I deleted the prior one and replaced it with the React-Leaflet implementation.